website/docs/reference/types/injections.mdx

```diff
@@ -1,15 +1,126 @@
+import SDKTabs from "@site/src/components/SDKTabs";
+import TabItem from "@theme/TabItem";
+
+
 # Injections
 
-## Raw
+Injection is a mechanism to get the value of a parameter from other sources than the graphql query.
+When a parameter is injected, no value is expected in the query, otherwise, the query will fail.
+They are specified at the type level for input types that are direct children of a `t.struct`.
+
+
+## Static
+
+**Description:**
+Static injection sets a static value to the parameter.
+
+**Method:** `.set(value)`
+
+**Parameter:** The value to be set, it must be compatible with the target type.
+
 
 ## Parent
 
-:::caution
+**Description:**
+Parent injection gets the value the output of a sibling field in the parent struct.
+It adds a dependency to the sibling field, so make sure to prevent circular dependencies.
```

Comment:
the **gets the value the ouput** sounds a bit confusing here, or am I the only one?